samza-test/src/test/java/org/apache/samza/test/framework/StreamApplicationIntegrationTest.java

```diff
@@ -76,7 +82,7 @@ public void testHighLevelApi() throws Exception {
         .addOverrideConfig("job.default.system", "test")
         .run(Duration.ofMillis(1500));
 
-    Assert.assertEquals(TestRunner.consumeStream(output, 10000).get(random.nextInt(count)).size(), 1);
+    Assert.assertEquals(TestRunner.consumeStream(output, Duration.ofMillis(1000)).get(random.nextInt(count)).size(), 1);
```

Comment:
Just double checking: Was it intentional to go from 10K ms to 1K ms?

Reply:
10K was a typo, this should not take more than 1K